src/pages/who-we-are/index.tsx

```diff
@@ -35,8 +41,68 @@ const EndSeparator = styled.div`
         border-bottom: 40px solid transparent;
     }
 `
+const target1 = '#start-separator'
+const options = {
+    root: null,
+    rootMargin: '0px',
+    threshold: 0.7,
+}
+
+const lazy_components: ReactNode = (
+    <>
+        <OurValues
+            fallback={
+                <div>
+                    <img src={VALUES_DUMMY} style={{ width: '95%' }} alt="our-values" />
```

Comment:
This Value changes a lot, we can't use this as dummy Image. I think, we should have few global dummy component, those can be used overall.  Also this is a plain HTML, why are we using dummy image for this?